source/future.tex

```diff
@@ -98,20 +98,64 @@
 prefix \tcode{++} and \tcode{\dcr} expressions\iref{expr.pre.incr}
 of \tcode{volatile}-qualified arithmetic and pointer types are deprecated.
 
+\begin{example}
+\begin{codeblock}
+volatile int velociraptor;
+++velociraptor;                 // deprecated: is this a single memory operation, two, or a six-foot turkey?
```

Comment:
The tone of this comment is too flippant, and doesn't really work as a rationale for deprecation (since the same question applies for all non-deprecated volatile operations, which are all just as implementation-defined as each other).

Just saying "// deprecated" seems fine (here and below). We don't need to justify why things are deprecated.

Reply:
It's a [very Cultured reference](https://www.youtube.com/watch?v=TbrsQNeEyIg), but I can drop it if you want.

Reply:
Quote from https://isocpp.org/std/standing-documents/sd-4-wg21-practices-and-procedures#code-of-conduct:
"Avoid metaphors, irony & be aware that jokes and humor may not translate"

Reply:
I dropped the reference, but I don't think we want to drop the explanations. I was explicitly asked for them in CWG.